src/Analyzers/MSTest.Analyzers.CodeFixes/AvoidExpectedExceptionAttributeFixer.cs

```diff
@@ -133,9 +133,8 @@ private static (SyntaxNode ExpressionOrStatement, SyntaxNode NodeToReplace)? Try
             {
                 return (expressionStatement.Expression, statement);
             }
-            else if (statement is LocalDeclarationStatementSyntax localDeclarationStatementSyntax &&
-                localDeclarationStatementSyntax.Declaration.Variables.Count == 1 &&
-                localDeclarationStatementSyntax.Declaration.Variables[0].Initializer is { } initializer)
+            else if (statement is LocalDeclarationStatementSyntax { Declaration.Variables.Count: 1 } localDeclarationStatementSyntax &&
+                     localDeclarationStatementSyntax.Declaration.Variables[0].Initializer is { } initializer)
```

Comment:
I think this can be simplified further to:

```suggestion
            else if (statement is LocalDeclarationStatementSyntax { Declaration.Variables: [{ Initializer: { } initializer }] })
```